src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue

```diff
@@ -50,7 +51,15 @@ export default {
 			customAppealEnabled: false,
 			globalBannerDenyList: [
 				'/checkout',
-				'/donate/support-kiva'
+				'/confirm-instant-donation/*',
+				'/donate/support-kiva',
+				'/error',
+				'/instant-donation-thanks',
+				'/join-team',
+				'/register/social',
+				'/possibility/giving-tuesday',
+				'/possibility/12-days-of-lending',
+				'/possibility/year-end'
```

Comment:
Thinking about moving this list to the new urlUtils file. Thoughts?

Reply:
+1 for putting this exclude list into urlUtils, seems like that's a good place for them to live.

Reply:
+1

Reply:
moved and amended